jinahub/indexers/cache/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jina==2.0.0rc5.dev54
```

Comment:
I believe we are at RC6 already? or even beyond. not sure.

Reply:
removed, using `master`